deploy/operator.yaml

```diff
@@ -32,3 +32,11 @@ spec:
               value: "mobile-developer-console-operator"
             - name: OPENSHIFT_HOST
               value: "${OPENSHIFT_HOST}"  # to be filled by whoever is deploying this file
+            - name: UPS_DOCUMENTATION_URL
+              value: ""
+            - name: IDM_DOCUMENTATION_URL
+              value: ""
+            - name: "SYNC_DOCUMENTATION_URL"
```

Comment:
Should `SYNC_DOCUMENTATION_URL` and `MSS_DOCUMENTATION_URL` not have double quotes around them like the other vars

Reply:
Good spot. They don't have to but it's best to keep consistent.